README.md

```diff
@@ -1,33 +1,103 @@
 # overseas-entities-web
-Web front-end for the Register An Overseas Entity service
 
-### Requirements
+Web front-end for the **register an overseas entity and tell us about its beneficial owners** service. Link to the live page [here](https://www.gov.uk/guidance/register-an-overseas-entity)
+</br>
+
+## Overseas entities architecture
+
+![Overseas entities architecture](./docs/Overseas%20entities%20architecture.png)
```

Comment:
In the diagram, maybe just depict CHIPS as a box rather than use the database icon - the box then represents CHIPS as a whole (Java web and services monolith, DB, Staffware etc.).